data/mupen64plus.ini

```diff
@@ -15907,27 +16208,35 @@ SaveType=None
 Mempak=Yes
 Rumble=Yes
 
-[C38ACBAE773CC3845EA354421E171998]
-GoodName=Turok 3 - Shadow of Oblivion (U) (Beta-WIP)
-CRC=D25C1211 13EEBF67
+[03D17AA3DC7663502017D3CC5A19AA8B]
+GoodName=Turok 3 - Shadow of Oblivion (E) (Beta) (2000-05-31)
+CRC=D124DBB7 7C9FF5E5
 Players=4
 SaveType=None
 Mempak=Yes
 Rumble=Yes
 
-[0A3A4C761960F7D648AFA60B1E565C7C]
-GoodName=Turok 3 - Shadow of Oblivion (U) (Beta) [h1C]
-CRC=D25C1211 13EEBF67
-RefMD5=69CE88C46A7C829C6F54004DE93EFCEF
-
 [69CE88C46A7C829C6F54004DE93EFCEF]
-GoodName=Turok 3 - Shadow of Oblivion (U) (Beta)
+GoodName=Turok 3 - Shadow of Oblivion (E) (Beta) (2000-06-06)
 CRC=37FA8F16 5F824D37
 Players=4
 SaveType=None
 Mempak=Yes
 Rumble=Yes
 
+[C38ACBAE773CC3845EA354421E171998]
+GoodName=Turok 3 - Shadow of Oblivion (E) (Beta) (2000-07-10)
+CRC=D25C1211 13EEBF67
+Players=4
+SaveType=None
+Mempak=Yes
+Rumble=Yes
```

Comment:
Replace 'Players', 'SaveType', 'Mempak' and 'Rumble' for: RefMD5=279EC83BD60A3CCE69A1DB22B0A5C318